### PR TITLE
[TIDY-FIRST] Fix isort for dbt-semantic-interfaces

### PR DESCRIPTION
**Pull request by QMalcolm**
resolves #N/A

### Problem

We've been grouping internal package imports by using `isort`. However, imports from `dbt_semantic_interfaces` weren't being grouped because I typo'd it as `dbt_semantic_interfaces` in the isort config.

### Solution

Fix the typo and rerun isort

### Checklist

- [X] I have read [the contributing guide](https://github.com/dbt-labs/dbt-core/blob/main/CONTRIBUTING.md) and understand what's expected of me  
- [X] I have run this code in development and it appears to resolve the stated issue  
- [X] This PR includes tests, or tests are not required/relevant for this PR
- [X] This PR has no interface changes (e.g. macros, cli, logs, json artifacts, config files, adapter interface, etc) or this PR has already received feedback and approval from Product or DX
- [X] This PR includes [type annotations](https://docs.python.org/3/library/typing.html) for new and modified functions

### CODE REVIEW

File: .isort.cfg [hunk 1]
```diff
@@ -1,4 +1,4 @@
 [settings]
 profile=black
 extend_skip_glob=.github/*,third-party-stubs/*,scripts/*
-known_first_party=dbt,dbt_adapters,dbt_common,dbt_extractor,dbt_semantic_interface
+known_first_party=dbt,dbt_adapters,dbt_common,dbt_extractor,dbt_semantic_interfaces
```
Comment (emmyoop) [inline .isort.cfg:4]: Should `snowplow_tracker` also be included here?  We maintain our own fork of it (minimal-snowplow-tracker)

**Review by emmyoop — CHANGES_REQUESTED** [review]

File: .isort.cfg (hunk 1 above)
Comment (QMalcolm) [reply .isort.cfg:4]: That's a good question 🤔 I do not know. We can, but it feels in a gray area. It is a fork that we maintain, that we don't appear to sync. However, it also overrides the name space of a non-internal package. If you think it's cut and dry that it should be included, happy to make the change. However, if you also feel that it's in a gray area, maybe it warrants more discussion and shouldn't be considered blocking for this PR.

Comment (emmyoop) [reply .isort.cfg:4]: After talking offline I agree we should solve this in #8409

**Comment by QMalcolm** [comment]
@cla-bot check

**Comment by QMalcolm** [comment]
Good bot

**Pull request merged**